src/date.rs

```diff
@@ -20,13 +20,15 @@ impl Date {
     const PLIST_EPOCH_UNIX_TIMESTAMP: Duration = Duration::from_secs(978_307_200);
 
     pub(crate) fn from_rfc3339(date: &str) -> Result<Self, ()> {
+        let offset: DateTime<FixedOffset> = DateTime::parse_from_rfc3339(date).map_err(|_| ())?;
         Ok(Date {
-            inner: humantime::parse_rfc3339(date).map_err(|_| ())?,
+            inner: offset.with_timezone(&Utc).into(),
         })
     }
 
     pub(crate) fn to_rfc3339(&self) -> String {
-        format!("{}", humantime::format_rfc3339(self.inner))
+        let datetime: DateTime<Utc> = self.inner.into();
+        datetime.to_rfc3339_opts(SecondsFormat::AutoSi, true)
```

Comment:
Should be SecondsFormat::Secs (https://github.com/opensource-apple/CF/blob/master/CFPropertyList.c#L1715).

We probably want a test that we are not using subsecond precision.